server/models/procedures/voucher.sql

```diff
@@ -220,4 +220,80 @@ BEGIN
   END IF;
 END $$
 
+
+-- this is handled on change rate changed
+DROP PROCEDURE IF EXISTS `UpdateLossAccount`$$
```

Comment:
Since this procedure is for Gain or Loss, and insert new entries in the journal could you rename it to something like : `GainOrLossForNewExchangeRate` it is a bit long 😄 but describe better what the procedure do.